code/modules/clothing/head/soft_caps.dm

```diff
@@ -202,12 +203,18 @@
 /obj/item/clothing/head/soft/propeller_hat/equipped(mob/living/user, slot)
 	. = ..()
 	if(slot & ITEM_SLOT_HEAD)
-		user.add_mood_event(PROPHAT_MOOD, /datum/mood_event/prophat)
+		user.add_mood_event(PROPHAT_MOOD, mood_event)
 
 /obj/item/clothing/head/soft/propeller_hat/dropped(mob/living/user)
 	. = ..()
 	user.clear_mood_event(PROPHAT_MOOD)
 	active = FALSE
 	update_icon()
 
+/obj/item/clothing/head/soft/propeller_hat/sad
```

Comment:
Would it be cleaner to just make the base-type have no mood event and put it on a subtype? Throw the base into the loadout, and have a `/happy` subtype to replace the single spot it can be found right now (the costumes vendor)

Reply:
doesnt matter